src/App.css

```diff
@@ -1,13 +1,47 @@
-nav {
+@import url('https://fonts.googleapis.com/css2?family=Montserrat&family=Roboto+Slab&display=swap');
+
+body {
+  background-color: #fafafa;
+}
+
+.navContainer {
   display: flex;
   flex-direction: row;
   justify-content: space-between;
+  position: relative;
   align-items: center;
+  font-family: 'Montserrat', sans-serif;
+  padding: 1% 5%;
+  background-color: #fff;
+  width: 90%;
+}
+
+#navTitle {
+  font-weight: bold;
+  color: #0290ff;
+  width: 15rem;
+  height: 2.313rem;
```

Comment:
Please ensure that you use the same `font attributes` as in the original design. To get the fonts or the color, click on the element you want to inspect and you will see the `right pan` like the image below:

![image](https://user-images.githubusercontent.com/10332499/163596665-917f9de4-5003-4e1e-8ed5-73b665b26b7c.png)